src/dropdown.js

```diff
@@ -21,10 +21,14 @@ export default class extends Controller {
 
   connect() {
     this.toggleClass = this.data.get('class') || 'hidden'
+    this.activeClass = this.data.get('activeClass') || ''
   }
 
   toggle() {
     this.menuTarget.classList.toggle(this.toggleClass)
+    if(this.activeClass != '') {
```

Comment:
Couldn't really think of a better way to check for null value here. If you have a suggestion on avoiding the conditional, I'm all ears.